app/src/main/java/me/ccrama/redditslide/Reddit.java

```diff
@@ -142,44 +130,16 @@ public static void forceRestart(Context context) {
         ProcessPhoenix.triggerRebirth(context, new Intent(context, MainActivity.class));
     }
 
-    public static void forceRestart(Context c, boolean forceLoadScreen) {

```

Comment:
I dropped this since the boolean was never actually used. If you passed in either true or false, it still did the same thing. I moved the boolean to the superior method that this was overloading and now you just have to pass in either true or false, which is fine.